Cargo.toml

```diff
@@ -14,11 +14,14 @@ panic = "abort"
 arrayvec = { version = "0.7.2", default-features = false }
 assertions = { path = "./assertions" }
 attestation = { path = "./attestation" }
+const-oid = { version = "0.9.0", features = ["db"] }
```

Comment:
Not related to this, but nit about commit message: "Each VM use" should be "Each VM uses"